### ISSUE TITLE
Some inputs missing programmatic labels

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#1634]
### Description

Some inputs are missing programmatic labels, which would assist users of assistive technology.

Examples include:
1. Text inputs where several entries per input are allowed (like contributor fields on the book info page). The select menu that contains the textbox has a label, but because the user lands on the text input when TABing through the page if at least one value has already been added, no label is communicated to ATs. 
1. `Chapter Type` and `Part` selection menus in the sidebar content on the 'Edit Chapter' page.
1. Many of the inputs on the theme options page are missing labels. This is particularly true when it comes to PDF options.  
1. All inputs on the Custom Styles page are missing labels.

While the inputs on the contributors page have programmatic labels, there are some mismatches between the value of the `for` attribute and the ID value of the input itself. For example, see https://github.com/pressbooks/pressbooks/blob/602e90ceab308df5d9629f86e43ea95c596bc09a/inc/admin/metaboxes/namespace.php#L1095-L1101. I believe that we can correct this by changing the for value to `contributor-first-name` and `contributor-last-name`.

**Comment by SteelWagstaff** [comment]
We can use WAVE's browser extensions to check for this systematically throughout the site: https://wave.webaim.org/extension/

**Comment by connerbw** [comment]
* Fixed inputs missing programmatic labels
* Changed to selectWoo, a fork of select2 with improved accessibility.
* Ajax load thema subject to improve speed when loading Book Info page.
* Fix errors reported by https://validator.w3.org/nu/

Remaining WAVE errors (obvious ones) come from TinyMCE and Wordpress Color Picker.  A "1 X Empty link" WAVE error comes from WordPress own code: `<a class='ab-item' href='#'><span class="ab-icon"></span><span class="screen-reader-text">Menu</span></a>` Won't fix for now.

**Comment by mcgratay** [comment]
Missing form label for Language & Script support selector on the Global Options page (Appearance > Themes Options); looks similar to other similar dropdown menus that have been fixed. 

Other pages have been improved but not completely resolved because of dependence on other software: TinyMCE, WordPress.
